### ISSUE TITLE
camera movement with right mouse button laggy

### ISSUE THREAD

**Issue by eXv1ru5** [Return-To-The-Roots/s25client#1584]
since one of the last updates, the camera movement with right mouse button is laggy.

**Comment by Xellzul** [comment]
Have you tried changing the refresh rate? Try unlimited or vsync if it changes anything

**Comment by eXv1ru5** [comment]
yes. the higher i set the framerate, the worse it gets. unlimited is worst just like vsync.
my only solution at the moment is to set the graphics driver to WinAPI in the graphics settings.

**Comment by Spikeone** [comment]
@eXv1ru5 is this still a valid issue? there was a bug regarding the configured framerate

**Comment by eXv1ru5** [comment]
@Spikeone yes, the problem still persists

**Comment by alfakr0ll** [comment]
@eXv1ru5 when I had a similar problem it helped delete the "config.ini"-file in the Return To The Roots save game folder. Unsure why that worked.
